shuffle/move/examples/main/mod.ts

```diff
@@ -78,55 +78,85 @@ export async function setMessageScript(
 // This is optional, as createTestNFTScriptFunction handles init.
 export async function initializeTestNFT() {
   const nftAddress = DiemHelpers.hexToAccountAddress(
-    defaultUserContext.address,
+      defaultUserContext.address,
   );
 
   // Create the type tag representing TestNFT to pass to the generic
   // script `initialize_nft`
   const testNftType = new DiemTypes.TypeTagVariantStruct(
-    new DiemTypes.StructTag(
-      nftAddress,
-      new DiemTypes.Identifier("TestNFT"),
-      new DiemTypes.Identifier("TestNFT"),
-      [],
-    ),
+      new DiemTypes.StructTag(
+          nftAddress,
+          new DiemTypes.Identifier("TestNFT"),
+          new DiemTypes.Identifier("TestNFT"),
+          [],
+      ),
   );
 
-  const script = codegen.Stdlib.encodeInitializeNftScript(testNftType);
-  const payload = new DiemTypes.TransactionPayloadVariantScript(script);
+  const payload = codegen.Stdlib.encodeInitializeNftCollectionScriptFunction(testNftType);
   return await DiemHelpers.buildAndSubmitTransaction(
-    defaultUserContext.address,
-    await devapi.sequenceNumber(),
-    await defaultUserContext.readPrivateKey(),
-    payload,
+      defaultUserContext.address,
+      await devapi.sequenceNumber(),
+      await defaultUserContext.readPrivateKey(),
+      payload,
   );
 }
 
-// ScriptFunction example; creation of NFT. Can only create one per account atm.
+// ScriptFunction example; creation of NFT.
 // See main/source/TestNFT.move
 export async function createTestNFTScriptFunction(
-  contentUri: string,
+    contentUri: string,
```

Comment:
For whatever reason, the formatting is off. Let's stick with deno's standard formatting, which can be run with `deno fmt`